### PR TITLE
refactor: Simplify environment configuration

### PR DESCRIPTION
**Pull request by irony**
This PR simplifies the environment configuration by: 
- Using a single .env file instead of multiple env files 
- Removing .env.production and .env.development 
- Updating configuration handling 
- Removing cross-env dependency

This change makes the configuration more straightforward and reduces complexity in the development workflow.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,11 +44,10 @@
     "zod": "^3.23.8"
   },
   "devDependencies": {
-    "@types/cors": "^2.8.17",
+    "@types/cors": "^2.8.17",
```
Comment (irony) [inline package.json:47]:
```suggestion
    "@types/cors": "^2.8.17",
```

File: package.json [hunk 2]
```diff
@@ -10,13 +10,13 @@
...
     "migrate": "npx prisma migrate deploy",
     "workers": "node --import tsx src/startWorkers.ts",
-    "dev-workers": "cross-env DOTENV_CONFIG_PATH=./.env.development node --import tsx --watch src/startWorkers.ts",
-    "dev-board": "cross-env DOTENV_CONFIG_PATH=./.env.development node --import tsx --watch src/index.ts",
+    "dev-workers": "node --import tsx --watch src/startWorkers.ts",
+    "dev-board": "node --import tsx --watch src/index.ts",
     "dev": "concurrently \"npm run dev-board\" \"npm run dev-workers\"",
-    "import": "node --import=tsx --env-file=.env.development scripts/import-spreadsheet-companies.ts",
-    "test": "DOTENV_CONFIG_PATH=./.env.test jest",
-    "prisma": "dotenv -e .env.development -- prisma",
```
Comment (Greenheart) [inline package.json:18]: We should remove the `dotenv-cli` dependency too since we no longer use the `dotenv` command.

File: package.json [hunk 3]
```diff
@@ -10,13 +10,13 @@
     "start": "node --import tsx src/index.ts",
     "migrate": "npx prisma migrate deploy",
     "workers": "node --import tsx src/startWorkers.ts",
-    "dev-workers": "cross-env DOTENV_CONFIG_PATH=./.env.development node --import tsx --watch src/startWorkers.ts",
-    "dev-board": "cross-env DOTENV_CONFIG_PATH=./.env.development node --import tsx --watch src/index.ts",
+    "dev-workers": "node --import tsx --watch src/startWorkers.ts",
+    "dev-board": "node --import tsx --watch src/index.ts",
     "dev": "concurrently \"npm run dev-board\" \"npm run dev-workers\"",
-    "import": "node --import=tsx --env-file=.env.development scripts/import-spreadsheet-companies.ts",
```
Comment (Greenheart) [inline package.json:16]: If we remove the env file from here, we need to add `import 'dotenv/config'` to the start of `scripts/import-spreadsheet-companies.ts`

**Review by Greenheart — APPROVED** [review]
Really nice to gather all environment specific config in one place!

**Pull request merged**